fission/src/ui/panels/configuring/assembly-config/interfaces/scoring/ZoneConfigInterface.tsx

```diff
@@ -190,14 +190,14 @@ const ZoneConfigInterface: React.FC<ZoneConfigProps> = ({ selectedField, selecte
         const fieldTransformation = JoltMat44_ThreeMatrix4(World.PhysicsSystem.GetBody(nodeBodyId).GetWorldTransform())
         const props = DeltaFieldTransforms_VisualProperties(deltaTransformation, fieldTransformation)
 
-        gizmo.mesh.position.set(props.translation.x, props.translation.y, props.translation.z)
-        gizmo.mesh.rotation.setFromQuaternion(props.rotation)
-        gizmo.mesh.scale.set(props.scale.x, props.scale.y, props.scale.z)
+        gizmo.obj.position.set(props.translation.x, props.translation.y, props.translation.z)
+        gizmo.obj.rotation.setFromQuaternion(props.rotation)
+        gizmo.obj.scale.set(props.scale.x, props.scale.y, props.scale.z)
```

Comment:
I don't think we should be directly mutating properties of the gizmo from outside.